agent-inject/agent/agent.go

```diff
@@ -4,33 +4,33 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"strconv"
-	"strings"
-
 	"github.com/mattbaird/jsonpatch"
 	corev1 "k8s.io/api/core/v1"
+	"strconv"
+	"strings"
```

Comment:
Hmm, generally we keep 3rd party packages separate from built-in:
```suggestion
	"strconv"
	"strings"

	"github.com/mattbaird/jsonpatch"
	corev1 "k8s.io/api/core/v1"
```

Reply:
yeah, sorry, Goland's fault =)